### ISSUE TITLE
MvcRouteListener not initialized in release 1.4.0

### ISSUE THREAD

**Issue by vincequeiroz** [zfcampus/zf-mvc-auth#117]
My application broke in release 1.4.0, because this code not exists anymore.

```
        $authentication = $this->container->get('authentication');
        $mvcAuthEvent   = new MvcAuthEvent(
            $mvcEvent,
            $authentication,
            $this->container->get('authorization')
        );
        $routeListener  = new MvcRouteListener(
            $mvcAuthEvent,
            $events,
            $authentication
        );
```
https://github.com/zfcampus/zf-mvc-auth/blob/1.3.2/Module.php#L86
https://github.com/zfcampus/zf-mvc-auth/blob/1.4.0/src/Module.php#L72

In documentation is not explicit that I must initialize the MvcAuthEvent and MvcRouteListener.

**Comment by lorenzofaresin** [comment]
+1 :(

**Comment by ksiondz-konstantin** [comment]
+1 ...

**Comment by webimpress** [comment]
Fixed in #120
